main.cpp

```diff
@@ -600,6 +732,22 @@ struct Fuselage
     void formsPlaneStructure (float fuselageDiameter = 234.3f);
     bool keepsHeatOut (std::string fuselageMaterial);
     void storeCargo (float);
+    void paintExterior(int paintWeight)
+    {
+        double mainDiameter = 1.112;
+        auto pw = paintWeight;
+        while (paintWeight < 600)
+        {
+            
+            for(pw = 1; pw < 3; ++pw)
+            {
+                mainDiameter += 1.11111;
+                std::cout << "More paint adds to a larger diameter of the fuselage." << std::endl;
+            }
+        break;      
+        }
```

Comment:
remove the `break`, because it prevents the `while` loop from looping.
- [ ] revise